src/servers/history/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM node:16-alpine AS BUILD_IMAGE
```

Comment:
Dockerfile should not be under `src` (its not source code).

Either on root level or if there are many images that need to be built in this repo under `image/<type>/Dockerfile`